### ISSUE TITLE
def _reward_feet_air_time(self):

### ISSUE THREAD

**Issue by 1874170538** [leggedrobotics/legged_gym#93]
def _reward_feet_air_time(self):
        # Reward long steps
        # Need to filter the contacts because the contact reporting of PhysX is unreliable on meshes
        contact = self.contact_forces[:, self.feet_indices, 2] > 1.
        contact_filt = torch.logical_or(contact, self.last_contacts) 
        self.last_contacts = contact
        first_contact = (self.feet_air_time > 0.) * contact_filt
        self.feet_air_time += self.dt
        rew_airTime = torch.sum((self.feet_air_time - 0.5) * first_contact, dim=1) # reward only on first contact with the ground
        rew_airTime *= torch.norm(self.commands[:, :2], dim=1) > 0.1 #no reward for zero command
        self.feet_air_time *= ~contact_filt
        return rew_airTime

This function will generate rewards both when it touches the ground and when it leaves the ground, not just when it touches the ground.

**Comment by 1874170538** [comment]
Hello 
I want to know if my understanding is correct.
